### PR TITLE
Add an entry on separating NPQ from ECF ✂️

### PR DESCRIPTION
**Pull request by peteryates**
Sorry this is way later than it should be.

### CODE REVIEW

File: app/posts/register-for-an-npq/2024-11-26-npq-becoming-standalone.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+The API and finance engines need NPQ data so everything the NPQ application collected was sent straight to the ECF application.
+
+![A flow chart showing registration data being fed into ECF by NPQ before being made available to training providers](/register-for-an-npq/2024-11-26-npq-becoming-standalone/npq-registration-data-flow.png)
+
+However, as time progressed built some fundamental differences emerged:
+
+1. NPQ registrations are made by teachers themselves but ECF registrations are done on behalf of the teacher by an administrator at the school.
+2. NPQ is elective but ECF is statutory.
+3. NPQ registration is a one-off transaction but ECF induction usually takes 2 years --- during that time information about the early career teacher's school and training needs to be kept up to date
+4. A teacher can obtain many NPQs but will only ever be an early career teacher once
```
Comment (claire-hughez) [inline app/posts/register-for-an-npq/2024-11-26-npq-becoming-standalone.md:35]:
```suggestion
1. NPQ registrations are made by teachers themselves but ECF registrations are done on behalf of the teacher by an administrator at the school
2. NPQ is elective but ECF is statutory
3. NPQ registration is a one-off transaction but ECF induction usually takes 2 years --- during that time information about the early career teacher's school and training needs to be kept up to date
4. A teacher can obtain many NPQs but will only ever be an early career teacher once
```

File: app/posts/register-for-an-npq/2024-11-26-npq-becoming-standalone.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+The NPQ and ECF registration functions are managed by different teams, and the API and finance engines are managed by a third team. We ended up with:
+
+* **two** policies
+* three teams
+* **two** code repositories
+* one database[^1]
+* one API
+* one finance engine
+
+This sharing of responsibilities means simple changes need to be carefully planned across teams and releases carefully coordinated. It also means that new NPQ features are reliant on the availability teams primarily working in the ECF space.
```
Comment (claire-hughez) [inline app/posts/register-for-an-npq/2024-11-26-npq-becoming-standalone.md:46]:
```suggestion
This sharing of responsibilities means simple changes need to be carefully planned across teams and releases carefully coordinated. It also means that new NPQ features are reliant on the availability of teams primarily working in the ECF space.
```

File: app/posts/register-for-an-npq/2024-11-26-npq-becoming-standalone.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+This is the riskiest part of the entire process. We knew it wouldn't be right first time, and because the accuracy of the data is vital we wanted confidence that it worked properly before doing it for real.
+
+We decided to use a process that:
+
+* could be run repeatedly
+* gave us clear feedback about what worked
+* output a list of things that didn't migrate cleanly and had to be investigated further
+
+In early November 2024 we hit a 100% success rate across all entities 🎉
```
Comment (claire-hughez) [inline app/posts/register-for-an-npq/2024-11-26-npq-becoming-standalone.md:103]:
```suggestion
In early November 2024 we hit a 100% success rate across all entities. 🎉
```

**Review by claire-hughez — COMMENTED** [review]
looks good to meee!

File: app/posts/register-for-an-npq/2024-11-26-npq-becoming-standalone.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+![NPQ API documentation](/register-for-an-npq/2024-11-26-npq-becoming-standalone/npq-api-docs.png)
+
+The new NPQ API very closely matches the ECF one so NPQ training providers will have a easier job in updating their integrations. The only significant changes are that endpoints in the old API that return combined ECF and NPQ data now only return NPQ data.
+
+Once the NPQ application is independent we will have:
+
+* **two** policies
+* ~~three~~ **two** teams
+* **two** code repositories
+* **two** databases
```
[the remark this answers is not in the document]
Comment (peteryates) [reply app/posts/register-for-an-npq/2024-11-26-npq-becoming-standalone.md:81]: Ah well spotted!

**Pull request merged**